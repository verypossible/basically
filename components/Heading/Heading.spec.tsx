import * as React from 'react'

import { shallow } from 'enzyme'
import 'jest-styled-components'

import { H1 } from './index'

describe('Heading', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<H1>Hi There</H1>)
    expect(tree).toMatchSnapshot()
  })
})
