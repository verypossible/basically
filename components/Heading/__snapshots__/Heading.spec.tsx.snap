// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading matches the snapshot 1`] = `
.c0 {
  color: #222;
  margin: 0;
  font-size: 1.5rem;
  margin: 0.5rem 0;
}

.c0 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: inherit;
}

@media (min-width:600px) {
  .c0 {
    font-size: 2rem;
  }
}

<h1
  className="c0"
>
  Hi There
</h1>
`;
